packages/forms-web-app/src/dynamic-forms/dynamic-components/task-list/index.njk

```diff
@@ -1,7 +1,5 @@
-{% extends "../../../views/layouts/lpa-dashboard/main.njk" %}
+{% extends layoutTemplate %}
 {% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
-
-{% set title="Appeal questionnaire - Manage appeals - GOV.UK" %}
 {% from "../../../views/macros/appeal-details.njk" import appealDetails %}
```

Comment:
Wouldn't the appeal-details macro need to go in the `questionnaire.njk` file where it's used?